tests/unit/test_network_connections.py

```diff
@@ -15,9 +16,12 @@
 
 sys.modules["ansible.module_utils.basic"] = mock.Mock()
 
-# pylint: disable=import-error, wrong-import-position
+try:
+    import network_lsr
+except ImportError:
+    sys.path.append(os.path.join(os.path.dirname(__file__), "fake_env"))
```

Comment:
Placing soft link in code folder is not a good choice to me.

Please create the link at runtime.

Reply:
Could you elaborate why is that not a good choice? This will naturally only run on Linux, and symlinks work there.

Writing files to the source tree while running a unit test does not seem a good choice either -- except in cases where the unit test needs to write files and has a special scratch directory for that.

Reply:
1. I have preference on clean code base. Having unnecessary soft link is not clean.
2. I used to face a loop softlink issue in other project, so I dislike softlink with `../../../../`.
3. Put it in runtime creation code could explains why this link is required.

Reply:
I don't know how to create the symlinks at test time in a way that is not bad.

We can close this MR and not merge it.